### ISSUE TITLE
New UI Importer Branch - Inspecting Selected Files spinner shows along top on CSV preview

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#395]
```
Using new UI importer branch,

1. Import any file that is at least 50 rows long.
2. Select the CSV/TSV preview.
3. scroll mouse all the way down.
4. div shows up incorrectly. (also appears that div at top with Google logo can 
bounce up and down with mouse scroll - weeee! ;)

<div class="dialog-frame dialog-busy" style=" "><div id="loading-message"><img 
src="images/large-spinner.gif"><span> Inspecting selected files 
...</span></div></div>

```

Original issue reported on code.google.com by `thadguidry` on 29 May 2011 at 2:57

Attachments:
- [Inspecting_Selected_Files.PNG](https://storage.googleapis.com/google-code-attachments/google-refine/issue-395/comment-0/Inspecting_Selected_Files.PNG)

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@gmail.com` on 27 Sep 2011 at 9:46
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified Fixed in Firefox 6 and Google Chrome 14.0.835.186 m
```

Original comment by `thadguidry` on 27 Sep 2011 at 3:05
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@google.com` on 9 Oct 2011 at 5:24
- Added labels: **Milestone-2.5**
